v3/spring-cloud-aws-v3-autoconfigure/pom.xml

```diff
@@ -10,8 +10,7 @@
         <version>2.3.0-RC2</version>
     </parent>
 
-    <groupId>io.awspring.cloud</groupId>
-    <artifactId>spring-cloud-aws-v3-autoconfigure</artifactId>
+	<artifactId>spring-cloud-aws-v3-autoconfigure</artifactId>
```

Comment:
review xml format here too

Reply:
Formatter does nothing to a file I am not sure what's happening. When I view the file it looks ok to me I am not sure why it is different here on Github.